src/components/DateTimePicker/Internal/ocpicker.module.scss

```diff
@@ -501,85 +676,315 @@
   &-year-partial,
   &-month-partial {
     .picker-cell-inner {
-      width: 56px;
+      width: 88px;
```

Comment:
why the magic number?

Reply:
This is the size of the picker-cell, not much magic here.

Reply:
Done